Automattic/Gravatar-SDK-Android
# QuickEditor: Media picker Popup

Closes #263

### Description

This PR implements QuickEditor styles Popup called `MediaPickerPopup` to display two options for uploading an image: `Choose a photo` and `Take photo`. 
The popup offset is set based on the `Upload image` button size.

Unfortunately, I haven't been able to create screenshot tests for `MediaPickerPopup`. For some reason, the popup is not drawn at all.


https://github.com/user-attachments/assets/0870ccf4-9b36-494b-8f5d-7e6746b36e3e



### Testing Steps

**Important Note**: The backend is currently returning the wrong format for the `updated date` field. As a workaround, you can apply this patch to comment that field from the Avatar object.
[Gravatar-Android-13-04-48.patch](https://github.com/user-attachments/files/16455284/Gravatar-Android-13-04-48.patch)

1. Make sure you have at least two avatars uploaded on Gravatar
2. Run the DemoApp 
3. Go to Avatar Update tab
4. Tap `Update Avatar` button
5. Follow the steps to log in (if necessary)
8. Tap `Upload Image` button
9. Confirm the media picker popup is visible
10. Tap on one of the options
11. Confirm the popup is dismissed
